test/regressions/regression-test-utils/download_vetoers.js

```diff
@@ -1,47 +1,76 @@
-const {blockscoutDownloadTokenHolders} = require('./blockscout_download_token_holders');
+const path = require("path");
+const {
+  blockscoutDownloadTokenHolders,
+} = require("./blockscout_download_token_holders");
+
+const NETWORK_NAME = "mainnet";
+const FILENAME_PREFIX = "";
 
 const ST_ETH_ADDRESS = "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84";
-const ST_ETH_HOLDERS_LOAD_CHUNKS_COUNT = 80;
-const ST_ETH_HOLDERS_FILE_NAME = "../complete-rage-quit-files/steth_vetoers.json";
+const ST_ETH_HOLDERS_LOAD_CHUNKS_COUNT = 3;
+const ST_ETH_HOLDERS_CHUNK_ADDRESSES_AMOUNT = 1000;
+const ST_ETH_HOLDERS_FILE_NAME = path.join(
+  __dirname,
+  "..",
+  "complete-rage-quit-files",
+  `${FILENAME_PREFIX}steth_vetoers.json`
+);
 const ST_ETH_HOLDERS_EXCLUDE_ADDRESSES = new Set([
-    "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0", // WstETH
-    "0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1" // WithdrawalQueue
+  "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0".toLowerCase(), // WstETH
+  "0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1".toLowerCase(), // WithdrawalQueue
```

Comment:
Maybe good add more Lido contracts that could contain steth

Reply:
Agreed, we should research which contracts hold stETH (e.g., InsuranceFund, Burner, Staking Modules, etc.). Let's add it in a standalone PR.